lib/puppet/module_tool/skeleton/templates/generator/README.md.erb

```diff
@@ -2,8 +2,7 @@
 
 #### Table of Contents
 
-1. [Overview](#overview)
-2. [Module Description - What the module does and why it is useful](#module-description)
+1. [Description](#description)
 3. [Setup - The basics of getting started with <%= metadata.name %>](#setup)
     * [What <%= metadata.name %> affects](#what-<%= metadata.name %>-affects)
```

Comment:
Adjust the following numbers? Not strictly required markdown-wise, but increases cognitive load to see the sequence 1,3,4,... ("is something wrong here?").

If the concern is number of lines changed, you could make them all `1.` and then never have this problem again :)